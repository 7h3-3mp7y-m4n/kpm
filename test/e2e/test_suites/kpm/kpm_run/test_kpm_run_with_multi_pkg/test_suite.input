kpm run kcl1/main.k kcl2/main.k