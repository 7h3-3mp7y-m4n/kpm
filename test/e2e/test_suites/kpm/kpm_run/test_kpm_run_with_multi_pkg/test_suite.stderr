error: only allows one package to be compiled at a time
error: cannot compile multiple packages [<workspace>/test_kpm_run_with_multi_pkg/kcl1 <workspace>/test_kpm_run_with_multi_pkg/kcl2] at the same time
